internal/suggestions/utils.go

```diff
@@ -0,0 +1,169 @@
+package suggestions
+
+import (
+	"fmt"
+	"github.com/manifoldco/promptui"
+	"gopkg.in/yaml.v2"
+	"path/filepath"
+	"regexp"
+	"strconv"
+	"strings"
+)
+
+func getLineNumber(errStr string) (int, error) {
+	lineStr := strings.Split(errStr, "[line ")
+	if len(lineStr) < 2 {
+		return 0, fmt.Errorf("line number cannot be found in err %s", errStr)
+	}
+
+	lineNumStr := strings.Split(lineStr[1], "]")[0]
+	lineNum, err := strconv.Atoi(lineNumStr)
+	if err != nil {
+		return 0, err
+	}
+
+	return lineNum, nil
+}
+
+func escapeString(input string) string {
+	re := regexp.MustCompile(`\\([abfnrtv\\'"])`)
+
+	// Replace escape sequences with their unescaped counterparts
+	output := re.ReplaceAllStringFunc(input, func(match string) string {
+		switch match {
+		case `\a`:
+			return "\a"
+		case `\b`:
+			return "\b"
+		case `\f`:
+			return "\f"
+		case `\n`:
+			return "\n"
+		case `\r`:
+			return "\r"
+		case `\t`:
+			return "\t"
+		case `\v`:
+			return "\v"
+		case `\\`:
+			return `\`
+		case `\'`:
+			return `'`
+		case `\"`:
+			return `"`
+		default:
+			// Should never happen, but just in case
+			return match
+		}
+	})
+
+	return output
+}
+
+func formatYaml(input string) (string, error) {
+	// Unmarshal the YAML input into a generic interface{}
+	var data interface{}
+	if err := yaml.Unmarshal([]byte(input), &data); err != nil {
+		return "", err
+	}
+
+	// Marshal the interface{} back to YAML with no extra spaces
+	output, err := yaml.Marshal(data)
+	if err != nil {
+		return "", err
+	}
+	outputStr := strings.TrimSpace(string(output))
+
+	return outputStr, nil
+}
+
+func acceptSuggestion() bool {
+	templates := &promptui.SelectTemplates{
+		Label:    "{{ . | cyan | bold }}",
+		Active:   "🐝 {{ .Emoji | yellow }} {{ .Name | yellow | bold }}",
+		Inactive: "   {{ .Emoji | white }} {{ .Name | white | bold }}",
+		Selected: "> {{ .Emoji | green }} {{ .Name | green | bold }}",
+	}
+
+	prompt := promptui.Select{
+		HideHelp: true,
+		Label:    "Would you like to accept the suggestion?",
```

Comment:
Just commenting here for organization's sake--let's change this to "press any key to continue"

Reply:
Yup will do. I'll make a separate branch for our write to file curated demo